### PR TITLE
V0.11.2 — Replacing babel ES2015 preset with env preset.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Changed
- Replaced babel ES2015 preset with env preset.

### CODE REVIEW

**Comment by DamianMullins** [comment]
We might need to update the browserlist to reflect our supported browsers, any idea where the latest list is @ashleynolan? 

The current list looks like this:

```json
"browserslist": [
   "> 5%",
  "last 2 versions",
  "ie > 8",
  "Safari >= 8"
]
```

**Comment by ashleynolan** [comment]
@DamianMullins I think this looks good tbh barring making the IE support `> 10` now.  Once you've changed it, would probably be worth making a note of the config and documenting it somewhere (like the wiki or global docs for now).

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Changelog
```
Comment (ashleynolan) [inline CHANGELOG.md]: 👏 Nice one

**Comment by DamianMullins** [comment]
I'll do the browserlist in another PR because there is a little bit of tweaking required in order for it to detect the correct browsers.

**Pull request merged**